tutorial/20_recipes/010_reevaluate_best_trial.py

```diff
@@ -0,0 +1,93 @@
+"""
+.. _reevaluate_best_trial:
+
+Re-evaluate the best trial
+==========================
+
+You can re-evaluate the objective with the best hyperparameters again
+after the hyperparameter optimization.
+For example,
+
+- You investigate deeply the behavior of ML model with the best hyperparameter.
+- You have optimized with Optuna using a partial dataset because the training time is long. After the hyperparameter tuning, you train the model using the whole dataset.
+
+Optuna provides interface to re-evaluate the objective function easily.
```

Comment:
```suggestion
Optuna provides an interface to re-evaluate the objective function using the current best values.
```

I like to avoid the words "easily" and "just." They can be intimidating for people who don't understand what you're doing.